docs/source/bibliography.bib

```diff
@@ -33,4 +33,14 @@ @misc{JuDo1000
     year = {2020},
     pages = {1-10},
     doi = {10.17605/OSF.IO/5ZPVK},
+}
+
+@article{SalvucciGoldberg2000,
+    author = {Salvucci, Dario D. and Goldberg, Joseph H.},
+    title = {Identifying Fixations and Saccades in Eye-Tracking Protocols},
+    journal = {Proceedings of the 2000 Symposium on Eye Tracking Research & Applications},
+    pages = {71–78},
+    year = {2000},
+    issn = {1581132808},
+    doi = {10.1145/355017.355028},
 }
```

Comment:
add a single new line to the end of the file.
it was missing somehow before but can be fixed with this PR.

Reply:
or don't worry, you can leave it as it is, i'll run pre-commit to reformat everything after successful merge anyhow